cargo-process.el

```diff
@@ -310,6 +310,7 @@ for error reporting."
 (defun cargo-process--tramp-file-name-prefix (file-name)
   "Return the TRAMP prefix for FILE-NAME.
 If FILE-NAME is not a TRAMP file, return an empty string."
+  (fboundp 'tramp-tramp-file-p)
```

Comment:
I'm not sure but shouldn't this be:

```Emacs Lisp
  (if (and (fboundp 'tramp-tramp-file-p)
           (tramp-tramp-file-p file-name)
        ,,,
```
As wel as for the one below?

Reply:
I appreciate your help. 
I have modified it according to the correct syntax.
thank you again.